src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -35,20 +35,23 @@ public static bool IsExpression(this IParameter parameter)
             CoreConstants.Parameters.REL_EXP_PLASMA, CoreConstants.Parameters.REL_EXP_VASC_ENDO);
       }
 
-      public static bool IsIndividualMolecule(this IParameter parameter)
+      public static bool IsExpressionOrOntogenyFactor(this IParameter parameter)
       {
          if (parameter.IsExpression())
             return true;
 
          if (CoreConstants.Parameters.OntogenyFactors.Contains(parameter.Name))
             return true;
 
-         if (parameter.IsNamed(CoreConstants.Parameters.REFERENCE_CONCENTRATION))
```

Comment:
I think the intent was right but the usage was wrong.
I have created two method `IsExpressionOrOntogenyFactor` and `IsIndividualMolecule`

The former `IsIndiviudalMolecule` implementation was I believe not OK